std_msgs/Header header
geometry_msgs/Point location
string sample_type
uint64 sample_idx
float64 scalar_value
